src/map/skill.cpp

```diff
@@ -12919,7 +12919,7 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		skill_area_temp[2] = 0;
 
 		if (sd && sd->weapontype1 == W_GATLING) {
-			i += 3;
+			i += 2;	// 11x11
```

Comment:
```suggestion
			i = 5; // 11x11
```
Should we put the absolute number here, to make it more clear?